ci_build/azure_pipelines/templates/unit_test.yml

```diff
@@ -14,3 +14,14 @@ steps:
     condition: succeededOrFailed()
     env:
         CI_ONNX_OPSET: '${{ onnx_opset }}'
+
+  - bash: |
+      export TF2ONNX_TEST_BACKEND=$CI_ONNX_BACKEND
```

Comment:
Our ci pipleline is already slow, don't think we want to add benchmarks to it.

Reply:
Fixed